casbah/src/main/scala/akka/contrib/persistence/mongodb/CasbahPersistenceJournaller.scala

```diff
@@ -34,11 +35,17 @@ class CasbahPersistenceJournaller(driver: CasbahMongoDriver) extends MongoPersis
     val batch = writes.map(write => Try(driver.serializeJournal(Atom[DBObject](write, driver.useLegacySerialization))))
     if (batch.forall(_.isSuccess)) {
       val bulk = journal.initializeOrderedBulkOperation
```

Comment:
Well, the write result is mapped to unit, agreed.  But a failure should flow through.